tools/floodgate/css/floodgate.css

```
@@ -12,3 +12,28 @@
   text-transform: uppercase;
   display: block;
 }
+
+.fg-modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.5);
```

Did you add stylelint as well? This should be:
```suggestion
  background-color: rgb(0 0 0 / 50%);
```